content/reference/wadm/usage.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Using wadm"
+date: 2022-09-08T11:02:05+06:00
+weight: 3
+draft: false
+---
+
+Using `wadm` typically involves using the `wash` command line tool. However, you can also use wash's supporting library in your Rust application or, if you continue to the next section, you'll see the API reference if you want to interact with wadm directly over a NATS connection.
+
+The following is an overview of the high level functionality exposed by `wadm`. To see the corresponding commands in the `wash` CLI, issue the following command:
+
+```
+wash app --help
+```
+
+### List Applications
```

Comment:
For this section, I think it would be really useful to have examples for each command using `wash app`